doc/release-notes-16528.md

```diff
@@ -0,0 +1,108 @@
+Wallet
+------
+
+### Experimental Descriptor Wallets
+
+Please note that Descriptor Wallets are still experimental and not all expected functionality
+is available. Additionally there may be some bugs and current functions may change in the future.
+
+0.21 introduces a new type of wallet - Native Descriptor Wallets. Descriptor Wallets store
```

Comment:
nit: I wonder if this can just be simplified from `Native Descriptor Wallets` to `Descriptor Wallets`

Reply:
Done.